frontend/src/components/IssueModeSelector/IssueModeSelector.tsx

```diff
@@ -0,0 +1,62 @@
+import { Box, Text } from '@highlight-run/ui/components'
+import clsx from 'clsx'
+
+import styles from './IssueModeSelector.module.css'
```

Comment:
Nit, but we are using [vanilla-extract](https://vanilla-extract.style/) for managing styles going forward. This is fine for now, but for future reference it would be best to use a `.css.ts` file rather than a CSS module file.